### DISCUSSION TITLE
Is a Look up value in Automation possible?

### DISCUSSION THREAD

**Discussion by Nasnl** [Budibase/budibase#5402] [General]
Is it possible to look up a value in a table in an automation based on a field, and then use that value in the next step (= sending an email)?

Ie, a new row is added to table A with value '1' in column AA. A lookup in table B for value '1' delivers a new value 'XX' and that is passed on to the next step (sending an email) with 'XX' in the body text.

Can't seem to figure this out...

**Comment by melohagan** [comment]
HI @Nasnl 

You could try using **Query Rows**.

So the flow of your automation may look as:

1. **Row Created** (For Table A)
2. **Query Rows**: (For Table B). Grab the value from Table A via `trigger.row.AA` to be used as a binding in the filter. 
3. **Send Email**: Grab the matching value found in the previous query with the binding: `{{ steps.1.rows.0.XX }}`

**Comment by Nasnl** [comment]
Thanks for your quick response @melohagan . I did not master the handlebars yet; so I had to upgrade my knowledge to solve this. It worked.
